### PR TITLE
Feature/push button

### LINKED ISSUE #5: Feature push-button
MVP for the input device. Registers a button push

### PR DESCRIPTION
**Pull request by rarriens**
Closes #5 
Created a Button class. Whenever a button is pressed, this will communicated via serial USB connection, visualize via Putty.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
+Button button(Pin_button);
+
+int button_state;
 
 void setup() 
 {
+  Serial.begin(Baudrate);
 }
 
 void loop() {
```
Comment (TimBuckers) [inline src/main.cpp:22]: the formatting is not consistent, the bracket in on the next line in the setup loop.

Is there a short-key for auto format that makes everything easier?

File: src/main.cpp [hunk 2]
```diff
@@ -1,18 +1,34 @@
...
- 
-  led.Blink(led_delay);
+
+  button_state = button.read_state();
+  
+  led.Blink(led_cycle);
+  
+  if (button_state == HIGH)
+    {
```
Comment (TimBuckers) [inline src/main.cpp]: this spacing also looks a bit weird (same as comment above use auto-format)

**Review by TimBuckers — APPROVED** [review]
Nice looks good!

File: src/main.cpp (hunk 1 above)
Comment (rarriens) [reply src/main.cpp:22]: Fixed by using auto formatting, Google C++ style

**Pull request merged**